### ISSUE TITLE
md syntax incompatible with a new metadata field

### ISSUE THREAD

**Issue by JamesKeny0n** [open-sdg/open-sdg#1165]
**Please complete the following information**
Data repository URL: github.com/Defra-Data-Science-Centre-of-Excellence/OIF-Dashboard-Data
Data or metadata issue: Metadata
Input format: YAML (Markdown)
**Describe the bug**

I've been creating new metadata fields in the prose.yml file and can successfully add plaintext to them.
A problem arises when I want to add plaintext and a hyperlink. If I add plaintext and a hyperlink (using md syntax '\[]()') the metadata field is not populated with any of the text I enter, rather the word 'Link'. The word 'Link' displays as a hyperlink in the metafield and if clicked will take you to an error webpage.

I can work around this issue by hijacking a built-in metadata field (using a translation key to change the title to a more suitable name). I can add text and a hyperlink and it will display on the site as intended.

**The bug I think I've encountered is that when custom metafields are created they haven't got md functionality.**
I think it exists because as far as I can tell, custom metafields weren't intended to be used and rather just the built-in fields and translation keys.

**Additional context**

**Comment by LucyGwilliamAdmin** [comment]
Hi @JamesKeny0n

I think the repo has been made private now so I'm unable to see what exactly you are trying to add. Where on the site does the metadata field show? Markdownify has been used on some parts of the site but not on others so just trying to understand if this is the case.

A similar case we have as default is Link to UN metadata but this actually requires two metadata fields - see here for example:
https://github.com/open-sdg/open-sdg-simple-starter/blob/develop/_prose.yml#L74-L86

Alternatively, an option could be to use html syntax:
`<a href="url">link text</a>`

**Comment by JamesKeny0n** [comment]
![metafield](https://user-images.githubusercontent.com/69935098/112303458-35c3b980-8c94-11eb-95ea-1079abdd6022.png)
I've attached a picture, the metafield in question is the 'Readiness and links to data'.

Under the hood this is the metafield 'national_indicator_description:' with a translation key to make it appear as 'Readiness and links to data'.
It is populated by indicator metadata - for  example the indicator featured in the picture contains this text:
```
national_indicator_description: 'This indicator is not available for reporting in 2020 in a finalised form. An interim indicator is presented here that shows riverine inputs of selected metals and nutrients into English tidal waters. Some data for this interim indicator are published annually ([RID data](https://oap.ospar.org/en/ospar-assessments/intermediate-assessment-2017/pressures-human-activities/contaminants/)) and the [EA emissions inventory](https://s3-eu-west- 1.amazonaws.com/data.defra.gov.uk/WaterQuality/wfd/Emission_Inventory_Chemical_Trends.zip) is published every six years. Further development is required to present trends for selected contaminants in an indicator. Contact the Environment Agency’s National Customer Contact Centre ([enquiries@environment-agency.gov.uk](mailto:enquiries@environment-agency.gov.uk)) for the OSPAR database extract used for the interim indicator.'

```


The bug I've had is that when I create my own metafield e.g. 'readiness_data' the metafield itself behaves in a different manner.

If I enter the same text block as above but for the 'readiness_data' metafield the metafield will be populated with just the word 'Link'.

This isn't a problem necessarily as the workaround I'm using works perfectly well, it's just strange that custom metafields don't hold the same functionality as the built-in ones.

On the UN metadata fields - I don't think they're really suitable as my understanding is that all of the text entered into the field becomes a hyperlink whereas we require some of the text to be plaintext.

Html syntax might work but I've yet to test it.

**Comment by LucyGwilliamAdmin** [comment]
Thanks for the further info @JamesKeny0n and for raising this issue - definitely strange that custom fields aren't working in the same way.

Are you able to post the full block for both fields from the prose files so I can try to recreate please?

**Comment by JamesKeny0n** [comment]
Code I added to the indicator itself:
```
national_indicator_description: 'This indicator is not available for reporting in 2020 in a finalised form. An interim indicator is presented here that shows riverine inputs of selected metals and nutrients into English tidal waters. Some data for this interim indicator are published annually ([RID data](https://oap.ospar.org/en/ospar-assessments/intermediate-assessment-2017/pressures-human-activities/contaminants/)) and the [EA emissions inventory](https://s3-eu-west-1.amazonaws.com/data.defra.gov.uk/WaterQuality/wfd/Emission_Inventory_Chemical_Trends.zip) is published every six years. Further development is required to present trends for selected contaminants in an indicator. Contact the Environment Agency’s National Customer Contact Centre ([enquiries@environment-agency.gov.uk](mailto:enquiries@environment-agency.gov.uk)) for the OSPAR database extract used for the interim indicator.'

readiness_data: 'This indicator is not available for reporting in 2020 in a finalised form. An interim indicator is presented here that shows riverine inputs of selected metals and nutrients into English tidal waters. Some data for this interim indicator are published annually ([RID data](https://oap.ospar.org/en/ospar-assessments/intermediate-assessment-2017/pressures-human-activities/contaminants/)) and the [EA emissions inventory](https://s3-eu-west-1.amazonaws.com/data.defra.gov.uk/WaterQuality/wfd/Emission_Inventory_Chemical_Trends.zip) is published every six years. Further development is required to present trends for selected contaminants in an indicator. Contact the Environment Agency’s National Customer Contact Centre ([enquiries@environment-agency.gov.uk](mailto:enquiries@environment-agency.gov.uk)) for the OSPAR database extract used for the interim indicator.'


```


Code I used in the prose.yml file:
```
- name: "national_indicator_description"
        field:
            element: text
            label: "Readiness and links to data"
            translation_key: tgLabels.national_indicator_description
            scope: national
			
			
- name: "readiness_data"
        field:
            element: text
            label: "Readiness and links to data"
            translation_key: tgLabels.readiness_data
            scope: national


```

Worth noting that you should probably delete my translation keys if you test it.

Thanks @LucyGwilliamAdmin for following this up!

**Comment by JamesKeny0n** [comment]
Hi @brockfanning @LucyGwilliamAdmin 

I made sure that this was certainly happening before I opened the issue .
However after reinserting the same code that I removed today, it now works as expected.
I now can't myself replicate this anomaly despite being stuck with it yesterday and this morning.

I've directly compared the commits I've made and they are identical - But the behaviour on the site is different - I can't explain what happened but thanks for all your help - If I figure what was causing it I'll let you know - I'll close this issue now.
